src/ethereum_test_forks/base_fork.py

```diff
@@ -4,8 +4,32 @@
 from abc import ABC, ABCMeta, abstractmethod
 from typing import Any, ClassVar, List, Mapping, Optional, Protocol, Type
 
+from semver import Version
+
 from .base_decorators import prefer_transition_to_method
 
+SOLC_FORKS_0_8_20 = [
+    "homestead",
+    "tangerineWhistle",
+    "spuriousDragon",
+    "byzantium",
+    "constantinople",
+    "petersburg",
+    "istanbul",
+    "berlin",
+    "london",
+    "paris",
+    "shanghai",
+]
+SOLC_VERSION_TO_SUPPORTED_FORKS = {
+    Version.parse("0.8.20"): SOLC_FORKS_0_8_20,
+    Version.parse("0.8.21"): SOLC_FORKS_0_8_20,
+    Version.parse("0.8.22"): SOLC_FORKS_0_8_20,
+    Version.parse("0.8.23"): SOLC_FORKS_0_8_20,
+    # TBA: 0.8.24 is the current dev version.
+    Version.parse("0.8.24"): SOLC_FORKS_0_8_20,
+}
```

Comment:
Would have preferred to have this in `ethereum_test_tools.code.yul`. I tried and had an issue with circular imports, wasn't sure how best to resolve it.